templates/base_index.html

```diff
@@ -22,49 +22,54 @@
 {% endblock body_class %}
 
 {% block outer_content %}
+
   {% block content %}
 
-    <!-- Test takeover template -->
-    <!-- {% block test_takeover_content %}
-    {% with title_1="A CTO's guide", title_2="to real-time Linux", subtitle='Understanding real-time systems, their use
-    cases and inner workings.', button_url="/", button_text="Download now",
-    img_url="https://assets.ubuntu.com/v1/fb1ea84e-Kernelt%20industries@2x.png", img_width="508", img_height="364" %}
-    {% include "takeovers/_test_template.html" %}
-    {% endwith %}
-    {% endblock test_takeover_content %} -->
-    <!-- Default Takeover: Download the latest Ubuntu -->
-    <section data-lang="all" id="takeover" class="p-section--hero">
-      <div class="row p-takeover-animation" id="takeover-animaition">
-        <div class="col-6">
-          <div class="p-section--shallow">
-            <h1 id="takeover-title" class="u-no-margin--bottom">What's new in Ubuntu {{ releases.latest.short_version }}?</h1>
-            <p class="p-heading--2" id="takeover-subtitle">
-              Introducing the Ubuntu Desktop for Raspberry Pi, the latest desktop features and micro clouds.
-            </p>
+    {% if request.cookies.get("control_or_variant") == 'variant' %}
```

Comment:
Just a thought, but as logic is server side, if it is a user's first visit to the site they wont be in either group when the takeovers are rendered

Reply:
as far as I can see from the code and from my trial - they will as the script for setting a cookie is run on loading the page

Reply:
![image](https://github.com/canonical/ubuntu.com/assets/58276363/0b295e1e-cc1e-48dd-b124-a5d251242f4b)